compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/platform/BackingDomInput.kt

```diff
@@ -54,9 +54,16 @@ internal class BackingDomInput(
     }
 
     fun focus() {
-        window.requestAnimationFrame {
+        // we focusing in next frame as a part of fix for https://youtrack.jetbrains.com/issue/CMP-7836/
```

Comment:
Is "focusing in next frame" still correct comment? I see that we removed window.requestAnimationFrame. Is it now about that we request in a handler that is also in the next frame?